tools/compatgen/internal/tensorflow.go

```diff
@@ -45,7 +51,7 @@ func FetchTensorFlowCompatibilityMatrix() ([]config.TFCompatibility, error) {
 	}
 
 	// sanity check
-	if len(compats) < 21 {
+	if len(compats) < 12 {
```

Comment:
With min_cuda_version, I am not sure what is best way to preserve the intention of this check so I am leaving as the floor value.